File: dd-java-agent/instrumentation/testng/src/testFixtures/java/org/example/TestSucceedUnskippable.java
package org.example;

import static org.testng.Assert.assertTrue;

import datadog.trace.api.civisibility.CIConstants;
import org.testng.annotations.Test;

@Test(groups = CIConstants.Tags.ITR_UNSKIPPABLE_TAG)
public class TestSucceedUnskippable {

  @Test
  public void test_succeed() {
    assertTrue(true);
  }
}

File: dd-java-agent/instrumentation/scalatest/src/test/scala/org/example/TestSucceedUnskippable.scala
package org.example

import datadog.trace.api.civisibility.CIConstants.Tags
import datadog.trace.api.civisibility.{CIConstants, InstrumentationBridge}
import org.scalatest.Tag
import org.scalatest.flatspec.AnyFlatSpec

object ItrUnskippableTag extends Tag(Tags.ITR_UNSKIPPABLE_TAG)

class TestSucceedUnskippable extends AnyFlatSpec {
  "test" should "assert something" taggedAs ItrUnskippableTag in {
    assert(2 + 2 > 3)
  }
}
